src/locale/misc/smartSearch/en.yml

```diff
@@ -47,6 +48,29 @@ most_active:
     examples:
         one: Add the 100 most active members of the organization before today.
         two: Remove the 5 most active members of the organization at any point in time.
+person_data:
+    inputString: "{addRemoveSelect} any person whose {criteria}."
+    ellipsis: "..."
+    addRemoveSelect:
+        add: Add
+        sub: remove
+    examples:
+        one: Add any person whose first name is 'Clara' and whose last name is 'Zetkin'.
+        two: Remove any person whose city is Stockholm.
+criteria:
+    criterion: "{field} matches {value}"
+    tuple: "{first} and {second}"
+    criteriaSelect:
+        first_name: first name
+        last_name: last name
+        gender: gender
+        co_address: co-address
+        city: city
+        zip_code: zip code
+        street_address: street address
+        alt_phone: alternative phone number
+        phone: phone number
+        remove: Remove this criteria
```

Comment:
Is this correct English? This/these criteria/criterion?

Reply:
Technically, criteria is plural and criterion is singular, but as a linguist I would argue -- sort of correct, yes -- 
https://www.merriam-webster.com/words-at-play/criteria-vs-criterion-singular-plural-grammar#:~:text=Criteria%20is%20typically%20a%20plural,used%20singulars%20datum%20and%20agendum.